app/assets/stylesheets/self-service-time-to-pay.css

```diff
@@ -376,4 +376,8 @@ details summary:focus,
 
 .details-margin {
     margin-bottom:20px;
+}
+
+span.keeptogether {
```

Comment:
you could call it ".nowrap" or ".no-wrap" maybe?

Reply:
done